products/opensuse/main.pm

```diff
@@ -257,6 +257,9 @@ sub load_inst_tests {
             loadtest "installation/livecd_network_settings";
         }
         loadtest "installation/partitioning";
+        if (get_var "ISO_IN_EXTERNAL_DRIVE") {
```

Comment:
The norm of this file seems to be to have `(` after method name instead of ` `, stick to this, please.